.github/workflows/wheels.yml

```diff
@@ -40,7 +40,7 @@ jobs:
           for cu in CU_VERSIONS[1:]:
             PY_CU.append((PY_VERSIONS[-1], cu))
           print("Limited matrix PY_CU", PY_CU)
-          PY_ROCM = [(PY_VERSIONS[-1], ROCM_VERSIONS[-1])]
+          PY_ROCM = [(PY_VERSIONS[-1], rocm_v) for rocm_v in ROCM_VERSIONS]
```

Comment:
This is fine for testing, but can we revert this before landing? This is to save on CI cost on PRs